corehq/apps/app_manager/models.py

```diff
@@ -986,9 +986,71 @@ def validate(self, value, required=True):
 
 
 class NavMenuItemMediaMixin(DocumentSchema):
+    """
+        Language-specific icon and audio.
+        Properties are map of lang-code to filepath
+
+        Convention (for old docs): Unless a specific language key exists,
+        by default, media for that language is same as that for English
+
+    """
+    media_image = SchemaDictProperty(JRResourceProperty)
+    media_audio = SchemaDictProperty(JRResourceProperty)
+
+    @classmethod
+    def wrap(cls, data):
+        self = super(NavMenuItemMediaMixin, cls).wrap(data)
+
+        # ToDo - Remove after migration
+        for media_attr in ('media_image', 'media_audio'):
```

Comment:
I could get that list by accessing attributes on cls, but if someone adds an extra attribute latter this would break, so I wrote it in a tuple. Also this code will be removed after the initial migration.